### DISCUSSION TITLE
Developing the project on MacOS

### DISCUSSION THREAD

**Discussion by hamini** [cagnulein/qdomyos-zwift#2416] [General]
I followed the installation instructions, but it didn't build for macos (m2), for example qmdnsengine dependency and Qmutex dependency aren't working (maybe it needs the latest version of qt5).

**Comment by cagnulein** [comment]
Yes you need qt 5.15.2

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 28 giu 2024 alle 20:55 hamini ***@***.***> ha
scritto:

**Comment by hamini** [comment]
Thank you, it works with this version.

**Comment by cagnulein** [comment]
great! feel free to open PRs with your development

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 29 giu 2024 alle 22:17 hamini ***@***.***> ha
scritto:
